app/controllers/discover.js

```diff
@@ -436,7 +448,7 @@ export default ApplicationController.extend({
             }
             this.set('start', '');
             this.set('end', '');
-            this.set('sort', '');
+            this.set('date', 'date_modified');
```

Comment:
Yeah, there was a discussion about what a user would expect to happen when clearing the filters and sorting was the odd one out. We concluded that sorting should not be reset when clearing filters, do you disagree?